app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -183,6 +184,94 @@ class RevisionCardFragmentTest {
     Intents.release()
   }
 
+  @Test
+  fun testRevisionCard_previousSubtopicTitle_hasCorrectContentDescription() {
+    launch<RevisionCardActivity>(
+      createRevisionCardActivityIntent(
+        context,
+        profileId.internalId,
+        "test_topic_id_0",
+        subtopicId = 1,
+        FRACTIONS_SUBTOPIC_LIST_SIZE
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.prev_subtopic_title)).check(
+        matches(
+          ViewMatchers.withContentDescription(
+            "The previous subtopic is "
+          )
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testRevisionCard_nextSubtopicTitle_hasCorrectContentDescription() {
+    launch<RevisionCardActivity>(
+      createRevisionCardActivityIntent(
+        context,
+        profileId.internalId,
+        "test_topic_id_0",
+        subtopicId = 1,
+        FRACTIONS_SUBTOPIC_LIST_SIZE
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.next_subtopic_title)).check(
+        matches(
+          ViewMatchers.withContentDescription(
+            "The next subtopic is "
```

Comment:
@Akshatkamboj14, how come this sentence and the one for the test above are incomplete?

Reply:
@adhiamboperes when I ran them, the test activity that were opening doesn't have any text on the textviews(prev_subtopic_title and next_subtopic_title) so only these outcomes were passing on my local.

Reply:
It means that those stories only have one chapters. 

In my opinion, you can safely remove these tests since they are the same as the fraction topic tests below.

Reply:
Hi @Akshatkamboj14, this is still an outstanding comment, please address it.

Reply:
Okay, I'll remove them